src/Wordpress/WordPressAPI.php

```diff
@@ -53,7 +53,11 @@ public function getHostAPIKey() { return null; }
      * @param null $userId
      * @return mixed
      */
-    public function getDomainList($userId = null) { return null; }
+    public function getDomainList($userId = null) 
```

Comment:
This will only ever return one domain, lets rename it to getDomain() or something that emphasizes the fact that it doesn't return a list.